types/angular-aria/angular-aria-tests.ts

```diff
@@ -0,0 +1,7 @@
+function testConfig($aria: angular.aria.IAriaService): void {
+    // $ExpectType boolean
+    $aria.config('tabindex');
```

Comment:
IMO this is not how that api is expected to be used:
https://docs.angularjs.org/api/ngAria/provider/$ariaProvider#overview

Reply:
This is not the provider.

Reply:
Let me find you some working code on our application that uses $aria typed as any.

```
function clickDirectiveLinkFunction($aria: any): angular.IDirectiveLinkFn {
  return ($scope, $element, $attrs) => {
    const runCallback = ($event: JQueryEventObject, $isMiddleClick = false) => {
      // Evaluates "my-click" expression with $event and $isMiddleClick as
      // locals.
      $scope.$eval($attrs['myClick'], {$event, $isMiddleClick});
    };

    $element.on('click', runCallback);
    $element.on('mouseup', ($event) => {
      if ($event.which === 2) {
        // A `which` value of 2 indicates a middle click.
        // See https://api.jquery.com/event.which/ for more details.
        runCallback($event, true);
      }
    });

    if ($aria.config('bindKeydown')) {
      $element.on('keydown', ($event) => {
        if ($event.key === 'Enter') {
          runCallback($event);
        }
      });
    }

    if ($aria.config('tabindex') && $element.attr('tabindex') === undefined) {
      $element.attr('tabindex', '0');
    }
  };
}
```

I'm fairly certain that it's a public service, otherwise I'm not sure why there would be an `@ngDoc` annotation.